styles/components/hero.scss

```diff
@@ -0,0 +1,31 @@
+.flex {
+  min-width: 20rem;
+  padding: 5rem;
+  background: $c-flex;
+
+  &__text {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    color: $c-white;
+    font-weight: bold;
+  }
+}
+
+.absolute {
+  position: relative;
+  width: 100%;
+  min-width: 20rem;
+  padding: 5rem;
```

Comment:
Like the other hero area, remove this padding and then set min-height and this will help see where there's some issues with the `top` positioning of the child element.